test/modules/core/shaderlib/shaderlib-test-utils.js

```diff
@@ -4,7 +4,16 @@ import Compiler from 'glsl-transpiler';
 const normalizeSource = source =>
   source
     // prepr (GLSL preprocessor) does not like #define without value
-    .replace(/^(#define \w+) *$/gm, ($0, $1) => `${$1} 1`);
+    .replace(/^(#define \w+) *$/gm, ($0, $1) => `${$1} 1`)
+    // remove comments due to glsl-transpiler issue #39
+    .replace(/(\/\*(.|\s)*?\*\/)|(\/\/.*)/g, '')
+    // remove unecessary exponential notation due to glsl-transpiler issue #41
+    .replace(/e\+00/g, '')
+    // work around glsl-transpiler issue #40
+    .replace(
+      /(\w+\[\d\])\.x = (\S+);\s*(\w+\[\d\])\.y = (\S+);/g,
+      ($0, $1, $2, $3, $4) => `${$1} = vec2(${$3}, ${$4});`
```

Comment:
I believe some of these issues are resolved.

Reply:
Don't see a new release yet, will update once it is available.